firefox-ios/Client/Configuration/version.xcconfig

```diff
@@ -1 +1 @@
-APP_VERSION = 139.0
+APP_VERSION = 140.0
```

Comment:
Do we want this change? I think it makes sense because is popping in my branches too, but just to confirm

Reply:
There was a conversion about this on Slack. It's not a problem to commit the change.